test/MicrosoftTeams.spec.ts

```diff
@@ -1972,6 +1972,26 @@ describe("MicrosoftTeams", () => {
     });
   });
 
+  // query test
+  describe("sendBotRequest", () => {
+    it("should not allow calls before initialization", () => {
+      expect(() =>
+      microsoftTeams.sendBotRequest({ query: "" }, () => {
+        return;
+      })
+    ).toThrowError("The library has not yet been initialized");
+    });
+    it("should allow calls after initialization", () => {
+      initializeWithContext("content");
+      expect(() =>
+      microsoftTeams.sendBotRequest({ query: "" }, () => {
+        return;
+      })
+    ).not.toThrowError();
+    });
+
```

Comment:
Consider adding some more unit test cases to check that:
1. The query is correctly sent to the parent (i.e. Teams)
2. The response correctly invokes the callback with the right data

Reply:
Will work on this in a separate PR

Reply:
Without corresponding unit tests it's generally difficult to ensure your code is actually doing what it's supposed to. In a true test-driven development process it's actually the unit tests that get written first:
https://en.wikipedia.org/wiki/Test-driven_development

As a best practice, I would urge you to add a full set of unit tests as part of your current PR.